docs/labels.md

```diff
@@ -0,0 +1,106 @@
+# GitHub Label List
+
+We use GitHub labels to perform issue triage, track and report on development
+progress, plan roadmaps, and automate issue grooming.
+
+To ensure that contributing new issues and PRs remains straight-forward, we would
+like to keep the labels required for submission to a minimum. The remaining
+labels will be added either by automation or manual grooming by other
+contributors and maintainers.
+
+This labels in this list originated within Kubernetes at
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md.
+
+## Labels that apply to issues or PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| api-review                         | Categorizes an issue or PR as actively needing an API review. | Any |
+| area/api                           | Issues or PRs related to an API | Any |
+| area/blog                          | Issues or PRs related to blog entries. | Any |
+| area/conformance                   | Issues or PRs related to conformance testing | Any |
+| area/dependency                    | Issues or PRs related to dependency changes | Any |
+| area/e2e-test                      | Issues or PRs related to end-to-end testing. | Any |
+| area/endpoint/identity             | Issues or PRs related to endpoint identity | Any |
+| area/endpoint/selection            | Issues or PRs related to endpoint selection | Any |
+| area/endpoint/type                 | Issues or PRs related to endpoint type | Any |
+| area/ipam                          | Issues or PRs related to IP address management (IPAM) | Any |
+| area/licensing                     | Issues or PRs related to Antrea licensing | Any |
+| area/monitoring/logging            | Issues or PRs related to logging | Any |
+| area/monitoring/auditing           | Issues or PRs related to auditing | Any |
+| area/monitoring/traffic-analysis   | Issues or PRs related to traffic analysis | Any |
+| area/monitoring/mirroring          | Issues or PRs related to mirroring | Any |
```

Comment:
So you would add areas we do not support yet?

Reply:
Yes, so I can track inbound issues and backlog items. This is handy for sorting issues and PRs as well. This taxonomy is not subject to change by every contributor and I'd like to establish upfront so that new issues have an appropriate area to land in during triage.

Reply:
I am just not sure if we might end up with many labels that will never be used. There are two cases - 1) something we do not have, but might confuse people we have or want to do? 2) something (code or feature) we do have, but no one uses the label because it is too detailed or they like another categorization (e.g. component vs feature).

Reply:
Added component areas as necessary.